nemo/collections/asr/models/aed_multitask_models.py

```diff
@@ -603,50 +578,33 @@ def forward(
 
         return transf_log_probs, encoded_len, enc_states, enc_mask
 
-    def compute_loss(
-        self, batch: tuple[torch.Tensor, torch.Tensor, torch.Tensor, torch.Tensor] | None
-    ) -> torch.Tensor:
-        """
-        Run forward pass through the model and compute the loss.
-
-        Args:
-            batch: a tuple of 4 tensors (signal, signal_len, tokens, tokens_len) as returned
-                by :class:`~nemo.collections.asr.data.audio_to_text_lhotse_prompted.PromptedAudioToTextLhotseDataset`.
-                When batch is ``None``, we'll return a zero tensor.
-        Returns:
-            The computed loss value as a single-element tensor.
-        """
+    # PTL-specific methods
+    def training_step(self, batch, batch_nb):
```

Comment:
add doc string for training step

Reply:
I can but this would be the only place in the module where we actually have training step doc strings.